docs/zh_cn/02-how-to-run/useful_tools.md

```diff
@@ -0,0 +1,198 @@
+- `checkpoint` : The path of the model checkpoint file.
+- `img` : The path of the image file used to convert the model.
+- `--work-dir` : Directory to save output ONNX models Default is `./work-dir`.
+- `--device` : The device used for conversion. If not specified, it will be set to `cpu`.
+- `--log-level` : To set log level which in `'CRITICAL', 'FATAL', 'ERROR', 'WARN', 'WARNING', 'INFO', 'DEBUG', 'NOTSET'`. If not specified, it will be set to `INFO`.
+
+## extract
+
+有 `Mark` 节点的 onnx 模型会被分成多个子图，这个工具用来提取 onnx 模型中的子图。
+
+### 用法
+
+```bash
+python tools/extract.py \
+    ${INPUT_MODEL} \
+    ${OUTPUT_MODEL} \
+    --start ${PARITION_START} \
+    --end ${PARITION_END} \
+    --log-level INFO
+```
+
+### 参数说明
+
+- `input_model` : The path of input ONNX model. The output ONNX model will be extracted from this model.
+- `output_model` : The path of output ONNX model.
+- `--start` : The start point of extracted model with format `<function_name>:<input/output>`. The `function_name` comes from the decorator `@mark`.
+- `--end` : The end point of extracted model with format `<function_name>:<input/output>`. The `function_name` comes from the decorator `@mark`.
+- `--log-level` : To set log level which in `'CRITICAL', 'FATAL', 'ERROR', 'WARN', 'WARNING', 'INFO', 'DEBUG', 'NOTSET'`. If not specified, it will be set to `INFO`.
+
+### 注意事项
+
+要支持模型分块，必须在 onnx 模型中添加  mark 节点，用`@mark` 修饰。
+下面这个例子里 mark 了 `multiclass_nms`，在 NMS 前设置 `end=multiclass_nms:input` 提取子图。
+
+```python
+@mark('multiclass_nms', inputs=['boxes', 'scores'], outputs=['dets', 'labels'])
+def multiclass_nms(*args, **kwargs):
+    """Wrapper function for `_multiclass_nms`."""
+```
+
+## onnx2pplnn
+
+这个工具可以把 onnx 模型转成 pplnn 格式。
+
+### 用法
+
+```bash
+python tools/onnx2pplnn.py \
+    ${ONNX_PATH} \
+    ${OUTPUT_PATH} \
+    --device cuda:0 \
+    --opt-shapes [224,224] \
+    --log-level INFO
+```
+
+### 参数说明
+
+- `onnx_path`: The path of the `ONNX` model to convert.
+- `output_path`: The converted `PPLNN` algorithm path in json format.
+- `device`: The device of the model during conversion.
+- `opt-shapes`: Optimal shapes for PPLNN optimization. The shape of each tensor should be wrap with "\[\]" or "()" and the shapes of tensors should be separated by ",".
+- `--log-level`: To set log level which in `'CRITICAL', 'FATAL', 'ERROR', 'WARN', 'WARNING', 'INFO', 'DEBUG', 'NOTSET'`. If not specified, it will be set to `INFO`.
+
+## onnx2tensorrt
+
+这个工具把 onnx 转成 trt .engine 格式。
+
+### 用法
+
+```bash
+python tools/onnx2tensorrt.py \
+    ${DEPLOY_CFG} \
+    ${ONNX_PATH} \
+    ${OUTPUT} \
+    --device-id 0 \
+    --log-level INFO
+```
+
+### 参数说明
+
+- `deploy_cfg` : The path of the deploy config file in MMDeploy codebase.
+- `onnx_path` : The ONNX model path to convert.
+- `output` : The path of output TensorRT engine.
+- `--device-id` : The device index, default to `0`.
+- `--calib-file` : The calibration data used to calibrate engine to int8.
```

Comment:
`--calib-file` is missed from the above `onnx2tensorrt`

Reply:
fixed it.